### ISSUE TITLE
Unity crashed while running mlagents-learn

### ISSUE THREAD

**Issue by canyon** [Unity-Technologies/ml-agents#3730]
I was following the instruction of [Learning-Environment-Create-New](https://github.com/Unity-Technologies/ml-agents/blob/0.15.1/docs/Learning-Environment-Create-New.md).

While I using the command `mlagents-learn config/config.yaml --run-id=RollarBall-1`, the window of Unity was closed, which happened a lot of times.

The step should reach 30000 which was set in the `config/config.yaml`, however, the program quit at the step of 10000.

Plus, the folder `models/RollarBall-1` is empty.

device: Windows 10 professional 1909
output:
(unity) C:\Users\Surface Pro 4\Desktop\2020Inno\2\ml-agents-0.15.1>mlagents-learn config/config.yaml --run-id=RollarBall-1

                        ▄▄▄▓▓▓▓
                   ╓▓▓▓▓▓▓█▓▓▓▓▓
              ,▄▄▄m▀▀▀'  ,▓▓▓▀▓▓▄                           ▓▓▓  ▓▓▌
            ▄▓▓▓▀'      ▄▓▓▀  ▓▓▓      ▄▄     ▄▄ ,▄▄ ▄▄▄▄   ,▄▄ ▄▓▓▌▄ ▄▄▄    ,▄▄
          ▄▓▓▓▀        ▄▓▓▀   ▐▓▓▌     ▓▓▌   ▐▓▓ ▐▓▓▓▀▀▀▓▓▌ ▓▓▓ ▀▓▓▌▀ ^▓▓▌  ╒▓▓▌
        ▄▓▓▓▓▓▄▄▄▄▄▄▄▄▓▓▓      ▓▀      ▓▓▌   ▐▓▓ ▐▓▓     ▓▓▓ ▓▓▓  ▓▓▌   ▐▓▓▄ ▓▓▌
        ▀▓▓▓▓▀▀▀▀▀▀▀▀▀▀▓▓▄     ▓▓      ▓▓▌   ▐▓▓ ▐▓▓    ▓▓▓ ▓▓▓  ▓▓▌    ▐▓▓▐▓▓
          ^█▓▓▓        ▀▓▓▄   ▐▓▓▌     ▓▓▓▓▄▓▓▓▓ ▐▓▓    ▓▓ ▓ ▓▓▓  ▓▓▓▄    ▓▓▓▓`
            '▀▓▓▓▄      ^▓▓▓  ▓▓▓       └▀▀▀▀ ▀▀ ^▀▀    `▀▀ ` ▀▀   '▀▀    ▐▓▓▌
               ▀▀▀▀▓▄▄▄   ▓▓▓▓▓▓,                                      ▓▓▓▓▀
                   `▀█▓▓▓▓▓▓▓▓▓▌
                        ¬`▀▀▀█▓

 Version information:
  ml-agents: 0.15.1,
  ml-agents-envs: 0.15.1,
  Communicator API: 0.15.0,
  TensorFlow: 1.7.1
C:\Anaconda3\envs\unity\lib\site-packages\tensorflow\python\framework\dtypes.py:517: 
2020-04-04 00:00:36 INFO [trainer_controller.py:167] Hyperparameters for the PPOTrainer of brain RollarBallBrain:
        trainer:        ppo
        batch_size:     10
        beta:   0.005
        buffer_size:    100
        epsilon:        0.2
        hidden_units:   128
        lambd:  0.95
        learning_rate:  0.0003
        learning_rate_schedule: linear
        max_steps:      3.0e4
        memory_size:    128
        normalize:      False
        num_epoch:      3
        num_layers:     2
        time_horizon:   64
        sequence_length:        64
        summary_freq:   10000
        use_recurrent:  False
        vis_encode_type:        simple
        reward_signals:
          extrinsic:
            strength:   1.0
            gamma:      0.99
        summary_path:   RollarBall-1_RollarBallBrain
        model_path:     ./models/RollarBall-1/RollarBallBrain
        keep_checkpoints:       5
2020-04-04 00:00:36.859589: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2020-04-04 00:04:57 INFO [trainer.py:214] RollarBall-1: RollarBallBrain: Step: 10000. Time Elapsed: 260.317 s Mean Reward: 0.196. Std of Reward: 0.397. Not Training.

(unity) C:\Users\Surface Pro 4\Desktop\2020Inno\2\ml-agents-0.15.1>

**Comment by awjuliani** [comment]
Hello @canyon 

Can you share what version of Unity you are running?

**Comment by canyon** [comment]
> Hello @canyon
> 
> Can you share what version of Unity you are running?

Unity 2018.4.19f1

**Comment by awjuliani** [comment]
Thanks for sharing these versions. Have you been able to make it all the way through this guide without Unity crashing: https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Getting-Started.md  ?

**Comment by canyon** [comment]
> Thanks for sharing these versions. Have you been able to make it all the way through this guide without Unity crashing: https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Getting-Started.md ?

Yes, I followed the instructions exactly. 
But now, after I clean up background programs in my computer, the problem was solved. 
I did some tests, and I think that is the reason causes Unity to crash on my computer. 
My CPU model is `Intel(R) Core(TM) i5-6300U CPU @ 2.40GHZ` and memory capacity is 8 GB.
Thank you for your help!

**Comment by awjuliani** [comment]
Glad you were able to resolve the issue.
